src/Cohesity.Powershell/Scripts/StorageDomain/Get-CohesityStorageDomain.ps1

```diff
@@ -0,0 +1,68 @@
+#### USAGE ####
+#   Get-CohesityStorageDomain
+#   Get-CohesityStorageDomain -Ids 37640784
+#   Get-CohesityStorageDomain -Names newmanStorage,newmanStorage-1576056989170 -FetchStats
+#   Get-CohesityStorageDomain -FetchStats
+###############
+
+function Get-CohesityStorageDomain {
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory = $false)]
+        [string[]]$Ids = $null,
+        [Parameter(Mandatory = $false)]
+        [string[]]$Names = $null,
+        [Parameter(Mandatory = $false)]
+        [string[]]$ClusterPartitionIds = $null,
```

Comment:
Its ok to get rid of ClusterPartitionIds

Reply:
Removed